indico/MaKaC/webinterface/tpls/CategModifMain.tpl

```diff
@@ -1,3 +1,5 @@
+${ render_template('flashed_messages.html') }
```

Comment:
It doesn't actually look **that** ugly. And `fixed_width=True` doesn't make it look that great either. But yeah, I'll add it for the sake of future usage.